react-forms/src/CheckBox.tsx

```diff
@@ -55,8 +55,8 @@ export default function CheckBox<T>(props: IProps<T>) {
             record[props.Field] = evt.target.checked as any;
             props.Setter(record);
           }}
-          value={props.Record[props.Field] != null ? 'on' : 'off'}
-          checked={props.Record[props.Field] != null ? true : false}
+          value={(props.Record[props.Field] as boolean) ? 'on' : 'off'}
```

Comment:
By casting props.Record[props.Field] to an unknown first before casting to a boolean fixes the lint/build errors.

Ex: `(props.Record[props.Field] as unknown as boolean)`